templates/questions/index.html.ep

```diff
@@ -5,6 +5,9 @@
 	<script src="/js/scripts.js"></script>
 	<link rel="stylesheet" media="screen and (max-device-width: 800px)" href="/css/style.css">
 	<link rel="stylesheet" media="screen and (min-device-width: 800px)" href="/css/desktop.css">
+	<% use Gravatar::URL; %>
+	<% my $user = $c->model->oauth2->find($c->session('id')); %>
+	<%my $gravatar = gravatar_url( email => $c->session->{email} , rating => 'pg'); %>
```

Comment:
Following the suggestion from the prior comment, this line would be

```
% my $gravatar = $self->gravatar->url( email => $c->session->{email} , rating => 'pg');
```

Notice I used a different [Mojo Template syntax](http://mojolicious.org/perldoc/Mojo/Template#SYNTAX).  Also be consistent with your spacing.  Notice the space after % on L9 but no space after % on L10.